src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="star stars--0">
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+    </div>
+    <div class="star stars--1">
```

Comment:
Add the blank line between the blocks
```suggestion
    </div>
    
    <div class="star stars--1">
```